_includes/projects/project-featured-in.html

```
@@ -20,18 +20,20 @@
     <div class="articles-scroll">
       {% for entry in related %}
         {% if entry.title %}
           {% comment %}Article entry — has a title, rendered via the existing article-item include{% endcomment %}
           {% assign utm_campaign = "project-" | append: project_slug %}
-          {% include articles/article-item.html article=entry utm_campaign=utm_campaign show_type_pill=true %}
+          {% assign item_anchor = "featured-in-" | append: entry.anchor %}
+          {% include articles/article-item.html article=entry utm_campaign=utm_campaign anchor=item_anchor show_share=true show_type_pill=true %}
         {% else %}
           {% comment %}Promotion entry — no title, rendered inline as a single-line post badge{% endcomment %}
-          <article class="article-item px-3 py-2">
+          <article class="article-item px-3 py-2"{% if entry.anchor %} id="featured-in-{{ entry.anchor }}"{% endif %}>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)